api/v1/vouchers/views.py

```diff
@@ -0,0 +1,23 @@
+import json
+from flask import request
+from ..api import api
+from ..decorators import limit
+from ..exceptions import VoucherException
+from models import Voucher
+
+__author__ = 'kenneth'
+
+
+@api.route('/voucher/validate', methods=['POST'])
+@limit(max_requests=10, group="voucher", by='ip')
```

Comment:
we can probably increase this `max_requests` limit quite a bit! maybe up to 1000 or 10000?